finance_dl/amazon.py

```diff
@@ -402,25 +402,21 @@ def invoice_link_finder(invoice_link):
                         href = "/".join(tokens)
                     return (order_id, href)
 
-                def invoice_link_finder_hidden():
-                        # submenu containing order summary takes some time to load after click
-                        # search for order summary link and compare order_id
-                        # repeat until order_id is different to last order_id
-                        summary_links = self.driver.find_elements(By.LINK_TEXT, 
-                            self.domain.order_summary)
-                        if summary_links:
-                            href = summary_links[0].get_attribute('href')
```

Comment:
What's the logic for inserting this code into directly the else clause? It seems more readable they way it was before.

Reply:
you are right, should be fixed now